en/developer_guide/api.rst

```diff
@@ -0,0 +1,112 @@
+API
+===
+
+Api access
+----------
+
+To manage the connection to the API, Open Orchestra uses Oauth2 protocol.
+The Oauth2 protocol provides to a client a secure access to API on behalf of a user.
+
+A client can be an application or a website for instance, he is represented by a pair key/secret.
+To use the API, the client must request a token.
+
+The token is a string that bound to the client and optionally to the user. It is used to authenticate the client
+when requesting a resource to API.
+
+
+Client
+------
+
+The client of API can be managed in the administration section of the back office.
+
+The creation of a client is done through the add form. In the form the pair key/secret is generated by the server.
+In this form you can specify if the client is credential.
+
+Token
+-----
+
+Open Orchestra provides two strategies for get a token.
+
+Client credentials
+~~~~~~~~~~~~~~~~~~
+
+This strategy allows to obtain a token only to at credential clients.
+
+For obtain it, send request :
+
+.. code-block::
+
+    /oauth/access_token?grant_type=client_credentials
+
+with HTTP header :
+
+.. code-block::
+
+    Authorization: Basic base64encode(key/secret)
```

Comment:
There was a mistake, you need to separate the key and the secret with a ``:`` instead